src/components/dark-mode-switcher/index.js

```diff
@@ -1,13 +1,14 @@
 import { memo, useCallback, useEffect } from "react";
 import { styled } from "@linaria/react";
+import PropTypes from "prop-types";
 
 import { Keys } from "utils/localStorage";
 import { useDarkMode } from "contexts/dark-mode-context";
 
 import DarkModeIcon from "assets/icons/dark-mode-switcher/dark-mode-symbol.svg";
 import LightModeIcon from "assets/icons/dark-mode-switcher/light-mode-symbol.svg";
 
-const DarkModeSwitcher = () => {
+const DarkModeSwitcher = ({ position }) => {
```

Comment:
why U need to add a position as a prop. Looks like, it will have only one value for all cases